web/src/components/molecules/Common/Form/utils.ts

```diff
@@ -23,3 +23,12 @@ export const keyReplace = (
   const { form, key } = formData;
   form.setFieldValue(key, e.currentTarget.value.replaceAll(" ", "-"));
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const emptyConvert = (value: any) => {
```

Comment:
Please use `unknown` instead of `any`
```suggestion
export const emptyConvert = (value: unknown) => {
```